java/core/src/main/java/sleeper/core/properties/instance/CommonProperty.java

```diff
@@ -358,7 +358,9 @@ public interface CommonProperty {
     UserDefinedInstanceProperty STATESTORE_COMMITTER_LAMBDA_CONCURRENCY_RESERVED = Index.propertyBuilder("sleeper.statestore.committer.concurrency.reserved")
             .defaultProperty(DEFAULT_LAMBDA_CONCURRENCY_RESERVED)
             .description("The reserved concurrency for the state store committer lambda.\n" +
+                    "Presently this value defaults to 10 to align with expectations around table efficiency. \n" +
                     "See reserved concurrency overview at: https://docs.aws.amazon.com/lambda/latest/dg/configuration-concurrency.html")
+            .defaultValue("10")
```

Comment:
This isn't being applied because the default property is still set. That needs to be removed.

Reply:
Addressed